atapas/webapis-playground
# [#127] Added a demo for Canvas API

# Description

Added a demo for the Canvas API as requested on the Issue. No new dependencies. 

Fixes #127 

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

It was tested locally multiple times. 
- Open the Canvas API demo
- Click on "Show Canvases"
- The canvases examples will appear below the button
- You can draw freely on the last example

# Checklist:

- [x] My code follows the [style guidelines](https://github.com/atapas/webapis-playground/blob/master/HOW-TO-ADD-DEMO.md) of this project
